mycore-solr/src/main/java/org/mycore/solr/proxy/MCRSolrProxyServlet.java

```diff
@@ -275,6 +276,15 @@ private void handleQuery(String queryHandlerPath, HttpServletRequest request, Ht
         solrHttpMethod.releaseConnection();
     }
 
+    private void filterParams(ModifiableSolrParams solrParameter) {
+        MCRConfiguration2.getString("MCR.Solr.Disallowed.Facets").ifPresent(df -> {
+            String[] disallowedFacets = df.split(" *, *");
```

Comment:
The parameter of split looks unusual.
We have also `MCRConfiguration2::splitValue` to process a comma-separated list of property-values.
There are lots of examples on how to use it in our code.